.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What does this file do?

Reply:
![image](https://user-images.githubusercontent.com/21251320/146261257-dcd4c1ae-23a0-44cd-8f00-2c7ac4b9b368.png)
it adds configuration for vscode's run and debug features

Reply:
Omggggg this makes sense! Thanks!